bin/index.js

```diff
@@ -70,6 +70,11 @@ new Command()
     '-t, --turn-off-link-shortening',
     "Use the full puml server link instead of the tiny url, if your diagrams are too big this won't work",
   )
+  .addOption(new Option(
+      '-l, --polr-url <url>',
+      "Use polr shortener instead tinyurl - optionally specify polr api key in POLR_APIKEY env var",
```

Comment:
From the code it would seem this API Key is required?

Reply:
polr api key is not mandatory, if specified in POLR_APIKEY env var then it will be used; else it will be sent as empty string in HTTP  request indicating request without api key. This works for polr installations that do not require api key.